dopamine/experiment5/experiment5_statstests.jl

```diff
@@ -0,0 +1,163 @@
+# # ───────────────────────────────────────────────────────────────────────────
+
+# # Is speed dependent on an interactions among all variables?
+# ftest(mwb.model, mwb_int.model)
+# # F-test: 2 models fitted on 625 observations
+# # ──────────────────────────────────────────────────────────────────────────
+# #      DOF  ΔDOF           SSR          ΔSSR      R²     ΔR²      F*   p(>F)
+# # ──────────────────────────────────────────────────────────────────────────
+# # [1]    6        2510861.8156                0.2551                        
+# # [2]   13     7  2375077.8791  -135783.9365  0.2954  0.0403  5.0065  <1e-04
+# # ──────────────────────────────────────────────────────────────────────────
+
+
+
+# ANOVAS
+
+# interaction between medium and worm
+mw = lm(@formula(meanspeed ~ medium + worm), trackstats)
+mw_int = lm(@formula(meanspeed ~ medium + worm + medium*worm), trackstats)
+
+ftest(mw.model, mw_int.model)
+# F-test: 2 models fitted on 625 observations
+# ─────────────────────────────────────────────────────────────────────────
+#      DOF  ΔDOF           SSR         ΔSSR      R²     ΔR²      F*   p(>F)
+# ─────────────────────────────────────────────────────────────────────────
+# [1]    5        3257210.6688               0.0337                        
+# [2]    7     2  3243550.4037  -13660.2651  0.0378  0.0041  1.3035  0.2723
+# ─────────────────────────────────────────────────────────────────────────
+
+# interaction between worm and bacteria 
+wb = lm(@formula(meanspeed ~ worm + bacteria), trackstats)
+wb_int = lm(@formula(meanspeed ~ worm + bacteria + worm*bacteria), trackstats)
+
+ftest(wb.model, wb_int.model)
+# F-test: 2 models fitted on 625 observations
+# ─────────────────────────────────────────────────────────────────────────
+#      DOF  ΔDOF           SSR         ΔSSR      R²     ΔR²      F*   p(>F)
+# ─────────────────────────────────────────────────────────────────────────
+# [1]    5        2592321.8552               0.2310                        
+# [2]    7     2  2525420.2672  -66901.5881  0.2508  0.0198  8.1990  0.0003
+# ─────────────────────────────────────────────────────────────────────────
+
+# interaction between worm and bacteria, also considering medium
+mwb = lm(@formula(meanspeed ~ medium + worm + bacteria), trackstats)
+wb_int_m = lm(@formula(meanspeed ~ medium + worm + bacteria + worm*bacteria), trackstats)
+
+ftest(mwb.model, wb_int_m.model)
+# F-test: 2 models fitted on 625 observations
+# ─────────────────────────────────────────────────────────────────────────
+#      DOF  ΔDOF           SSR         ΔSSR      R²     ΔR²      F*   p(>F)
+# ─────────────────────────────────────────────────────────────────────────
+# [1]    6        2510861.8156               0.2551                        
+# [2]    8     2  2452259.7245  -58602.0911  0.2725  0.0174  7.3842  0.0007
+# ─────────────────────────────────────────────────────────────────────────
```

Comment:
is this the GLM that you replicated with R?
if so, I got p = 0.0007 and you got 0.000677 if I am interpreting correctly, which seems good

Reply:
Okay great! Anything thing else I need to prepare before the meeting?